bin/desi_archive_tilenight

```diff
@@ -77,21 +113,27 @@ elif args.prod.count("/") == 0:
 else:
     reduxdir = args.prod
 
-if args.specstatus is not None:
-    log.info(f'Reading tiles from {args.specstatus}')
-    tiles = Table.read(args.specstatus)
-    keep = (tiles['SURVEY'] == 'main') & (tiles['ZDONE'] == 'true')
-    if args.night is not None:
-        log.info(f'Filtering to just LASTNIGHT={args.night}')
-        keep &= tiles['LASTNIGHT'] == args.night
-
-    tiles = tiles[keep]
-else:
-    tiles = Table(names=('TILEID', 'LASTNIGHT'), dtype=(int, int))
-    tiles.add_row( (args.tileid, args.night) )
+if args.specstatus is None:
+    args.specstatus = os.path.expandvars('$DESI_ROOT/survey/ops/surveyops/trunk/ops/tiles-specstatus.ecsv')
+    if not os.path.exists(args.specstatus):
```

Comment:
I don't think this file existence check should be under the outer "`if __ is None`". As it stands it wouldn't check if a user provided file exists. 
In either case `Table.read` will throw an error anyway, but I think that defeats the purpose of this snippet.

Reply:
Good point, I'll fix that.